src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Delete.cs

```diff
@@ -84,7 +84,7 @@ public static partial class HttpClientJsonExtensions
         /// <param name="client">The client used to send the request.</param>
         /// <param name="requestUri">The Uri the request is sent to.</param>
         /// <param name="type">The type of the object to deserialize to and return.</param>
-        /// <param name="context">Source generated JsonSerializerContext used to control the deserialization behavior.</param>
+        /// <param name="context">The JsonSerializerContext used to control the deserialization behavior.</param>
```

Comment:
Technically a `JsonSerializerContext` could have been manually extended by the user and not by a source generator, but I think the change is ok otherwise.